src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function(operation, number) {
+      operation.bind(this)(number);
```

Comment:
:+1: but you may use `.call` instead of `bind` if you want to call the function immediately
just FYI